test/orderings.jl

```diff
@@ -0,0 +1,26 @@
+@testset "Orderings" begin
+
+    import KnuthBendix.Alphabet, KnuthBendix.set_inversion!, KnuthBendix.Word
+    import KnuthBendix.LenLexOrder, Base.Order.lt
+
+    A = Alphabet{String}(["a", "b", "c", "d"])
```

Comment:
Based on a sibling pull request it will be enough to write
```julia
A = Alphabet(["a", "b", "c", "d"])
```
However, maybe it is enough to create an alphabet of chars? I would need to update it in my tests.
```julia
A = Alphabet(['a', 'b', 'c', 'd'])
```

Reply:
Certainly, it will be enough. I just copied that line from the test and modified slightly (not reflecting too much on that).